packages/cashier/src/pages/payment-agent/payment-agent-card/payment-agent-card.jsx

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { ExpansionPanel } from '@deriv/components';
+import PaymentAgentCardDescription from './payment-agent-card-description';
+import PaymentAgentCardDepositDetails from './paymen-agent-card-deposit-details';
+import './payment-agent-card.scss';
+
+const PaymentAgentCard = ({ payment_agent }) => {
```

Comment:
@george-usynin-binary Why not go with TS for newly added files? 🤔

Reply:
let’s hold off TS migration for now (c) Nijil